docs/concepts/services-networking/manging-hosts-file-with-host-aliases.md

```diff
@@ -0,0 +1,71 @@
+---
+assignees:
+- rickypai
+- thockin
+title: Manging Hosts File with HostAliases
+redirect_from:
+- "/docs/user-guide/manging-hosts-file-with-hostaliases/"
+- "/docs/user-guide/manging-hosts-file-with-hostaliases.html"
+---
+
+* TOC
+{:toc}
```

Comment:
Could we make the documentation more direct so users do not need to read the entire page to understand how to use the feature?

----

## Usage
Users can add custom entries to their pod `/etc/hosts` with the `HostAliases` field in a pod spec. 

Note that the contents of `/etc/hosts` in a container is managed automatically by Kubelet and overwritten on pod creation/restart, thus custom modifications to the file will not persist.

*example*
...
*output /etc/hosts*
...

## Limitations
...

## Discussion

Put the context about Docker etc here